Triangle.js

```diff
@@ -0,0 +1,16 @@
+// Задача с построением треугольника
+
+const stairs = () => {
+    let string = '';
+
+    const buildStairs = () => {
+        while(string.length < 7) {
+            console.log(string += '#')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Varlyle/my-first-project/pullRequest?prid=7306921)
```suggestion
            console.log(string += "#")
```